frontend/src/views/ProprietairePage.vue

```diff
@@ -632,6 +632,7 @@ export default defineComponent({
           Vous pouvez aussi l'obtenir sur ce
```

Comment:
Je propose cette modification pour que le message soit plus clair.
Modifier également le mot "site" du lien par "societe.com"

```suggestion
          Vous pouvez aussi l'obtenir sur le site
```